federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/wfs/WfsSourceUtils.java

```diff
@@ -54,118 +52,69 @@
             "http://%s:%d/services/wfs",
             "http://%s:%d/wfs");
 
-    public UrlAvailability confirmEndpointUrl(WfsSourceConfiguration config) {
-        Optional<UrlAvailability> result = URL_FORMATS.stream()
-                .map(formatUrl -> String.format(formatUrl,
-                        config.sourceHostName(),
-                        config.sourcePort()))
-                .map(url -> getUrlAvailability(url, config.sourceUserName(), config.sourceUserPassword()))
-                .filter(avail -> avail.isAvailable() || avail.isCertError())
-                .findFirst();
-        return result.isPresent() ? result.get() : null;
+    private static String wfsVersionExp = "/wfs:WFS_Capabilities/attribute::version";
+
+    public static ProbeReport sendWfsCapabilitiesRequest(String url, String username, String password) {
```

Comment:
Can we leave javadoc on all theses methods that says what the success, warning, failure and return types are so we don't have to dig through the method to see what it returns?